### PR TITLE
Assignment7+8

### PR DESCRIPTION
**Pull request by xiokd**
Partner: @dhoxie

### CODE REVIEW

**Comment by NicholasJohnsonDev** [comment]
I'll review this.

File: Assignment/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <link rel="icon" type="image/png" sizes="32x32" href="favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="favicon/favicon-16x16.png">
+    <link rel="manifest" href="favicon/site.webmanifest">
+
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+</head>
+
+    <body class="wrapper">
+        <header class="header">
+            <div class="ewu"">EWU</div>
```
Comment (NicholasJohnsonDev) [inline Assignment/index.html:16]: Line 16: extra quote mark
<div class="ewu"">EWU</div>

**Comment by NicholasJohnsonDev** [comment]
Looks very nice. I regret not using flex as you did. 

To fix:
Your buttons are a little greener in their gradient, but I doubt that matters.
"Another Joke" button should have white font.
No delay between the setup and punchline.
No hover effects on buttons.
Extra credit not done.

General
The project should look like the image below ✔
The project should have an index.html file ✔
With an index.js file in a js folder ✔
And a site.css file in a css folder (along with other css files) ✔
There should be a <head>
With a favicon ✔
And a title ✔
The <body> should use the middle 70% of the page ✔

Menu Bar
The page should have a title/menu bar ✔
Use Font Awesome for the hamburger icon  ✔
When clicked, the menu should appear under it ✔
The menu should have four buttons with creative names (the buttons don't need to do anything) ✔

Create a card with <div>s and CSS
Create a card from <div>s ✔
The card should have a header, body, and footer ✔
The CSS for this should be in the CSS file and work easily for another card. ✔

Show jokes on the card
Use the joke API from the first assignment to display a joke on a card (https://v2.jokeapi.dev/joke/Programming) ✔
Use Axios from the CDN for the API call. If the joke API returns an error display a message to try again in a few moments ✔
The punch line should not show immediately, but appear after 4 seconds ❌
There should be a button to get another joke ✔
There should be <a> tag that links to the joke api documentation page ✔
This <a> should be manually styled as a button ✔
Buttons should have a background that is a gradient ✔
All buttons (even the <a> that you made into a button) should have a hover effect ❌

Extra Credit
Do the following:
Make the buttons in the menu do something creative ❌
Rather than just making the menu appear, have it fade in and slide down. ❌
Have the page jump to 100% width on a smaller screen and 50% on a larger screen with 70% on a regular screen.  ❌

Ensure that:
This is built with raw HTML, CSS, and JavaScript, no external libraries aside from Axios and a font library ✔
The index.html file should work when opened from a file explorer ✔
Keep all files in appropriate folders ✔
Have warning free HTML, CSS, and JavaScript ✔
Use as little css as possible ✔
No inline CSS ✔
Validate HTML with: https://validator.w3.org/nu/#textarea (or similar) ✔ -- There was only one error, so you probably did but just typoed after.
Validate CSS with: https://jigsaw.w3.org/css-validator/#validate_by_input (or similar) ✔
No errors when loading the page ✔
Choose simplicity over complexity ✔

File: Assignment/js/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    var type = jokeData.type;
+
+    if (type == 'single') {
+        // joke is a one-liner
+        displaySingleJoke(jokeData);
+    }
+    if (type == 'twopart') {
+        // joke with setup and delivery
+        displayFirstLine(jokeData);
+        setTimeout(displaySecondLine, 4000, jokeData);
```
Comment (mmwoodfo) [inline Assignment/js/index.js:19]: You should move this outside of your if statement

**Review by mmwoodfo — COMMENTED** [review]
Looks good!

**Pull request closed without merging**